vpr/src/route/route_timing.cpp

```diff
@@ -1669,6 +1675,107 @@ static void print_route_status(int itry, double elapsed_sec, float pres_fac, int
     fflush(stdout);
 }
 
+static void print_overused_nodes_status(const t_router_opts& router_opts, const OveruseInfo& overuse_info) {
+    //Display upper limit
+    if (int(overuse_info.overused_nodes) > router_opts.max_reported_overused_rr_nodes) {
```

Comment:
It seems that if the number of overused nodes is larger than the threshold, we will do nothing.
Should we still report the overuse status and leave a message that the listed nodes are limited by the thresholds set by users.

Reply:
I think we'll log the overuse info if the number is small as well as produce an overuse info report file.

Reply:
Thanks! That would be perfect.

Reply:
I assume you want all the nets at the congested node as well? Do you want the report to be indexed by the congested nets or the overused RR nodes?

Reply:
Yes. Showing all the congested nets will be helpful. I prefer to be indexed by the overused RR Nodes, where we can see what nets share the same the node.

Reply:
I've added the nets to the report. You can checkout the report by specifying **--generate_rr_node_overuse_report on**. I will deal with the strong Id issue soon.

Reply:
Adding the congested nets is a good idea.  In addition to printing their ids, I think we should also print their names and the type of driving block. E.g.
net #83 (processor|alu|reset) driven by a block of type CLB
as that can help locate them in the netlist and see patterns faster.

Reply:
Sorry, I forgot about printing more detailed info on the nets. Working on this right now